CONTRIBUTING.md

```
@@ -0,0 +1,271 @@
+  void exampleMethod();
+  
+private:
+  int m_exampleVariable;
+};
+```
+
+### Commit Messages
+
+Follow [Conventional Commits](https://www.conventionalcommits.org/):
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)